fleet/lib/kube-prometheus-stack/aggregator/values.yaml

```diff
@@ -6,8 +6,21 @@ defaultRules:
   create: true
   labels:
     lsst.io/rule: "true"
-  additionalRuleLabels:
-    receivers: ",squadcast,"
+  additionalGroupRuleLabels:
```

Comment:
I don't think this value exists. https://github.com/search?q=repo%3Aprometheus-community%2Fhelm-charts%20additionalGroupRuleLabels&type=code

Reply:
https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L103C3-L103C28 appearently I inversed some internally. But in the current version this become irrelevant. We keep deploying the prometheus rules from helm chart so they can be viewed in the alertmanager and prometheus overviews of the alerts, but none of these default rules should have the `receiver` label.

I'm a much larger fan of deploying the "real" alerts alongside with the applications.